config/application.rb

```diff
@@ -19,7 +19,12 @@ class Application < Rails::Application
     # Use the responders controller from the responders gem
     config.app_generators.scaffold_controller :responders_controller
 
-    config.load_defaults 7.0
+    config.load_defaults 7.1
+
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    config.autoload_lib(ignore: %w(assets tasks templates))
 
```

Comment:
I'm not sure if this means we can remove the `require` lines at the bottom of this file in the lib directory?

Reply:
I read somewhere for the new release, they autoload the `lib`.